src/python/bot/fuzzers/afl/engine.py

```diff
@@ -50,25 +73,55 @@ def fuzz(self, target_path, options, reproducers_dir, max_time):
     Args:
       target_path: Path to the target.
       options: The FuzzOptions object returned by prepare().
-      reproducers_dir: The directory to put reproducers in when crashes
-          are found.
+      reproducers_dir: The directory to put reproducers in when crashes are
+        found.
       max_time: Maximum allowed time for the fuzzing to run.
 
    Returns:
       A FuzzResult object.
     """
-    config = launcher.AFLConfig.from_target_path(target_path)
+    environment.set_value('AFL_ENGINE_TIMEOUT', max_time)
```

Comment:
rather than set an env var here to be used later, can we just modify runner's fuzz method to accept this argument, which is then passed to afl's get_fuzz_timeout?

Reply:
+1